dm/syncer/syncer.go

```diff
@@ -1517,8 +1528,15 @@ func (s *Syncer) Run(ctx context.Context) (err error) {
 	}
 	if cleanDumpFile {
 		tctx.L().Info("try to remove all dump files")
-		if err = os.RemoveAll(s.cfg.Dir); err != nil {
-			tctx.L().Warn("error when remove loaded dump folder", zap.String("data folder", s.cfg.Dir), zap.Error(err))
+		doNotClean := false
```

Comment:
why not just use failpoint to make cleanDumpFile= false?

Reply:
yes it can be cleanDumpFile= false. i will fix.